influxdb/client.py

```diff
@@ -84,6 +82,8 @@ def __init__(self,
         self.udp_port = udp_port
         if use_udp:
             self.udp_socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        else:
```

Comment:
For the tests to succeed:
remove the "else:" part and keep the `self._session = ..` always.